channeldb/invoice_test.go

```diff
@@ -690,11 +712,9 @@ func TestQueryInvoices(t *testing.T) {
 			t.Fatalf("unable to add invoice: %v", err)
 		}
 
-		// We'll only settle half of all invoices created.
-		if i%2 == 0 {
-			_, err := db.UpdateInvoice(
-				paymentHash, getUpdateInvoice(amt),
-			)
+		if i%3 == 0 {
+			// From every three invoices settle one.
+			_, err := db.UpdateInvoice(paymentHash, getUpdateInvoice(amt))
```

Comment:
Since we're allowing settled invoices to be considered expired as well, we should have a set of settled expired invoices.

Reply:
done